test/networkd-test.py

```diff
@@ -130,7 +130,7 @@ def assert_link_states(self, **kwargs):
 
         # Wait for the requested interfaces, but don't fail for them.
         subprocess.call([NETWORKD_WAIT_ONLINE, '--timeout=5'] +
-                        ['--interface=%s' % iface for iface in kwargs])
+                        ['--interface={}'.format(iface for iface in kwargs)])
```

Comment:
Hey, that's not right! You changes a list comprehension to a list with a single item like (`['--interface=<generator at 0x...>']`).

Reply:
Sorry i missed it. I just simply change position of ")"